Source/FORMDataSource.m

```diff
@@ -500,6 +503,70 @@ - (void)fieldCell:(UICollectionViewCell *)fieldCell updatedWithField:(FORMField
         self.configureFieldUpdatedBlock(fieldCell, field);
     }
 
+    NSArray *components = [field.fieldID componentsSeparatedByString:@"."];
```

Comment:
This is duplicated functionality, I'll do a refactoring before removing the WIP prefix